lib/portlayer/exec/config.go

```diff
@@ -38,9 +38,13 @@ type Configuration struct {
 
 	// Resource pools under which all containers will be created
 	ComputeResources []types.ManagedObjectReference `vic:"0.1" scope:"read-only"`
+
 	// Resource pool is the working version of the compute resource config
 	ResourcePool *object.ResourcePool
 
+	// uuid of portlayer vm
+	PortlayerUUID string
```

Comment:
I'm not clear what purpose this serves. I would imagine everything in this struct will be vsphere-specific, immutable and at the scope of the VCH. Is this so we can distinguish between container VMs created by different VCHs? Is the resource pool they're in not a reliable enough filter?